packages/react-adminer/src/components/List.tsx

```diff
@@ -110,9 +134,32 @@ export const List: React.FC<Props> = ({ entityName, filter = true }) => {
 			: (v: any, object: any) => {
 					if (object.id === activeRecord?.id && activeRecord?.property === f.name) {
 						if (!(f as any).creatable) {
-							return <div>nelze editovat</div>;
+							notification.error({
+								message: `It is forbidden to change the value in the column: ${f.name}`,
```

Comment:
To je sice správně, ale když se ti ta komponenta přerenderuje 20x, vyskočí ti 20notifikací. Takže by to bylo lepší vyřešit jinak. Zamyslíš se sám jak?